link/kprobe.go

```diff
@@ -49,6 +49,9 @@ type KprobeOptions struct {
 	// Can be used to insert kprobes at arbitrary offsets in kernel functions,
 	// e.g. in places where functions have been inlined.
 	Offset uint64
+	//Set kretprobe's MAXACTIVE if you need.
+	//Set to 0 (or absent) will make the kernel default to NR_CPUS.
+	Maxactive int
```

Comment:
Let's add some more documentation and also deprecate this API to dissuade people from using it unless really necessary.

```suggestion
	// Increase the maximum number of concurrent invocations of a kretprobe.
	// Required when tracing some long running functions in the kernel.
	//
	// Deprecated: this setting forces the use of an outdated kernel API and is not portable
	// across kernel versions.
	RetprobeMaxActive int
```